uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -471,4 +471,25 @@ public static function getEnabledAbsenceTypes()
 
     return $absenceTypes;
   }
+
+  /**
+   * Returns the AbsenceType where the must_take_public_holiday_as_leave option
+   * is set
+   *
+   * @return \CRM_HRLeaveAndAbsences_BAO_AbsenceType|null
+   */
+  public static function getOneWithMustTakePublicHolidayAsLeaveRequest() {
```

Comment:
mmm .. not sure what else a suitable name would be ... okay no problem you can leave it as is